### PR TITLE
Sea breath

### PR DESCRIPTION
**Pull request by Florie25**
Draft new Pulse post.
The .jpg image can be changed

### CODE REVIEW

File: _posts/2021-07-06-Subnero-SeaBreath-partnership.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: Subnero now present in Japan
+banner : images/banner-pulse-standard.jpg
+date:  2021-07-06
+categories: wnc
+thumbnail: images/pulse-seabreath.jpg
+---
+
+
+
+Subnero,  is delighted to add [SeaBreath](https://www.sea-breath.com/)to their list of business partner.
```
Comment (AnneThKu) [inline _posts/2021-07-06-Subnero-SeaBreath-partnership.md]:
```suggestion
Subnero,  is delighted to add [SeaBreath](https://www.sea-breath.com/) to their list of business partner.
```

File: _posts/2021-07-06-Subnero-SeaBreath-partnership.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Seabreath is based in Tokyo and will represent Subnero in Japan.
```
Comment (AnneThKu) [inline _posts/2021-07-06-Subnero-SeaBreath-partnership.md]:
```suggestion
SeaBreath is based in Tokyo and will represent Subnero in Japan.
```

**Review by AnneThKu — CHANGES_REQUESTED** [review]

**Comment by AnneThKu** [comment]
Screenshot of the post.
image

**Comment by notthetup** [comment]
Do we need a space between `SeaBreath` and `to`?

**Review by manuignatius — APPROVED** [review]

**Review by AnneThKu — APPROVED** [review]

**Pull request merged**